contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/BasicFetchStrategy.java

```diff
@@ -120,6 +122,14 @@ public BasicFetchStrategy(HdfsFetcher fetcher,
                 int read;
 
                 while (true) {
+                    if (status.hasException()) {
+                        Exception ex = status.getException();
+                        if (ex.getClass().equals(AsyncOperationStoppedException.class)) {
```

Comment:
should it be instance of AsyncOperationStoppedException instead of class name equals ?